components/ILIAS/Saml/README.md

```diff
@@ -145,6 +145,6 @@ The key `NameIDPolicy` cannot be a string. It must be an array of the following
 
 Due to the routing changes in ILIAS 10 the `baseurlpath` needs to be changed in the `$ILI_DATA/auth/saml/config/config.php`. New installations will have this value as default.
 ```diff
-- 'baseurlpath' => 'Services/Saml/lib/',
+- 'baseurlpath' => 'components/ILIAS/Saml/lib/',
```

Comment:
@mjansenDatabay Is this correct?

Reply:
Hi @klees ,

this is on our internal backlog since @lscharmer provided the change (to adopt the changes/correct path for ILIAS 9.x) to the ILIAS 10.x "README.md" with #8089 .

We will take care of this outside this PR (which has been already merged :-)).

Best regards,
Michael